.github/workflows/basic.yaml

```diff
@@ -12,7 +12,7 @@ name: Basic
 jobs:
   test:
     name: Test Suite
-    runs-on: ubuntu-8-core-32-gb
+    runs-on: blacksmith-32vcpu-ubuntu-2204
```

Comment:
is this 32 cores on purpose? everything else is 16

Reply:
The test action is the main bottleneck

@aayushshah15 Can you switch back to 16-core to see the performance again with caching?

Reply:
we decided to go with 32-cores for all actions for now @aayushshah15